.github/workflows/ci_workflow.yml

```diff
@@ -257,6 +257,9 @@ jobs:
                 --target install \
                 --config ${{ matrix.build-type }}
         working-directory: _build
+      - name: Validate
+        run: |
+          share/ci/scripts/linux/validate_imath_libs.sh _install 
```

Comment:
Should we be passing in the source dir here to validate the release notes? or split that out to a separate script?